### DISCUSSION TITLE
Display contents from an attached file.

### DISCUSSION THREAD

**Discussion by gmentzik** [Budibase/budibase#8625] [Help]
I want to store some txt type files (txt,csv,json)  as attachments in CouchDB (MinIO) using attachment field but then be able to display contents of the text file at the page or even more use it as input at at form's longtext field after I manipulate contents with javascript. 
For images there is a way to display images. Is there something similar for txt files?

**Comment by melohagan** [comment]
Hey @gmentzik 

You could take a look at accomplishing this with a custom component: https://docs.budibase.com/docs/custom-component

There's an example of a basic PDF reader here: https://github.com/YuanZhang98/budibase-pdf-viewer
